apps/console-electron/postcss.config.mjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('postcss-load-config').Config} */
+const config = {
+  plugins: {
```

Comment:
Hey, can we import tailwind config same way we did for playground? Would be better to have common tailwind base config

Reply:
I don't really understand tailwind yet, so not sure if this config will be enough to support theming.  I have a separate task for this in our [milestones](https://docs.google.com/spreadsheets/d/1zkySjaZR9BgdjjtdIFdLK_piWPAKC9qD9FVX9u8RrN0/edit?gid=0#gid=0)

Reply:
Cool, I’ll do that since i set it up in playground too, will be fadter